Presentation.Web/app/components/org/user/org-user-delete.controller.ts

```diff
@@ -242,12 +244,9 @@
             if (!confirm('Er du sikker på, at du vil slette brugeren?')) {
                 return;
             }
-            this.userRoleAdministrationService.removeUser(this.loggedInUser.currentOrganizationId, this.userToModify.Id)
-                .then(success => {
-                    if (success) {
-                        this.$uibModalInstance.close();
-                    }
-                });
+            
+            this.userService.deleteUser(this.userToModify.Id, this.loggedInUser.currentOrganizationId)
```

Comment:
why the switch... you already had it and that service you called before was about userroleadministration